src/resources/scss/components/_tooltip.scss

```diff
@@ -93,4 +94,8 @@
 .tooltip-preview {
     background: $white;
     color: $black;
+
+    & {
```

Comment:
I generally don't see anything wrong with it, I'm just wondering why we need a selector `.tooltip-preview .tooltip-preview`?

Reply:
Should be `img` instead of `&`